geode-core/src/main/java/org/apache/geode/management/internal/configuration/persisters/ConfigurationPersisterFactory.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.geode.management.internal.configuration.persisters;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.apache.geode.cache.configuration.CacheConfig;
+import org.apache.geode.cache.configuration.CacheElement;
+import org.apache.geode.cache.configuration.RegionConfig;
+
+public class ConfigurationPersisterFactory {
+  private final CacheConfig cacheConfig;
+  private final Map<Class, Class> persisterMap = new HashMap<>();
+
+  public ConfigurationPersisterFactory(CacheConfig cacheConfig) {
+    this.cacheConfig = cacheConfig;
+    persisterMap.put(RegionConfig.class, ClusterRegionConfigPersister.class);
```

Comment:
why would you not just instantiate the Persister? Better (and cheaper) than having to do a `Class.newInstance()` every time you want to use it